### PR TITLE
Add Images demo to the gallery app.

### PR DESCRIPTION
**Pull request by amirh**
We are still waiting for the final assets to use, for now I just used
some place holders.

### CODE REVIEW

File: examples/flutter_gallery/lib/gallery/item.dart [hunk 1]
```diff
@@ -306,6 +306,14 @@ List<GalleryItem> _buildGalleryItems() {
       routeName: CupertinoSwitchDemo.routeName,
       buildRoute: (BuildContext context) => new CupertinoSwitchDemo(),
     ),
+    // Media
+    new GalleryItem(
+      title: 'Images',
+      subtitle: 'Static and animated images',
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/gallery/item.dart]: This demo is just about animated images, so maybe adjust the title/subtitle

**Review by HansMuller — APPROVED** [review]
You'll need to update the demos lists in examples/flutter_gallery/test/live_smoketest.dart and examples/flutter_gallery/test_driver/transitions_perf_test.dart

**Comment by amirh** [comment]
Thanks, I addressed the comments.

I'm waiting for the final assets before submitting this.
@sethladd - FYI

File: examples/flutter_gallery/test/live_smoketest.dart [hunk 2]
```diff
@@ -100,6 +100,9 @@ const List<Demo> demos = const <Demo>[
   const Demo('Sliders'),
   const Demo('Switches'),
 
+  //Media
```
Comment (Hixie) [inline examples/flutter_gallery/test/live_smoketest.dart]: nit: missing space

Comment (amirh) [reply examples/flutter_gallery/test/live_smoketest.dart]: done

**Comment by HansMuller** [comment]
LGTM

**Pull request merged**